spec/models/case_study_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe CaseStudy do
+  let(:content_item) do
+    GovukSchemas::Example.find("case_study", example_name: "doing-business-in-spain")
+  end
+
+  it_behaves_like "it has updates", "case_study", "has-updates"
+  it_behaves_like "it has updates", "case_study", "only-has-first-published-at"
+  it_behaves_like "it has no updates", "case_study", "doing-business-in-spain"
+  it_behaves_like "it has no updates", "case_study", "no-public-updated-at"
+  it_behaves_like "it can have worldwide organisations", "case_study", "doing-business-in-spain"
+  it_behaves_like "it can have emphasised organisations", "case_study", "doing-business-in-spain"
+
+  describe "#contributors" do
+    it "returns the organisations ordered by emphasis" do
+      model_instance = described_class.new(content_item)
```

Comment:
To be consistent with other tests this variable should be called `content_item` rather than `model_instance`

e.g. 
```ruby
content_item = described_class.new(content_store_response)
```